### PR TITLE
Write the argument value back to input box when sharing url.

### PR DESCRIPTION
**Pull request by simozhan**
PDC-1401

### CODE REVIEW

**Comment by bliuredhat** [comment]
looks good for me

**Comment by simozhan** [comment]
Fix conflict

File: src/app.jsx [hunk 1]
```diff
@@ -364,11 +371,17 @@ var LoadForm = React.createClass({
...
         var roles = this.props.roles.map(function (val) {
-            return <option key={val}>{val}</option>;
+            return <option value={val}>{val}</option>;
         });
+        var component = (this.props.params['component']) ? this.props.params['component']:"";
+        $("#component").attr("value", component);
+        var release = (this.props.params['release']) ? this.props.params['release']:"all";
+        $("#release option[value='" + release + "']").attr("selected", "selected");
+        var role = (this.props.params['role']) ? this.props.params['role']:"all";
+        $("#role option[value='" + role + "']").attr('selected','selected');
```
Comment (ycheng-aa) [inline src/app.jsx]: Duplicate code. Should we use method to handle this three?

**Comment by ycheng-aa** [comment]
The function looks good to me.

File: src/app.jsx (hunk 1 above)
Comment (simozhan) [inline src/app.jsx]: component is a bit different with the other two.
the other two are abstracted in a common method.

**Comment by ycheng-aa** [comment]
:+1:

**Pull request merged**